app/views/mailers/reservation_mailer/car_reservation_approved.html.erb

```diff
@@ -23,10 +23,13 @@
           <li> <%= p %> </li>
         <% end %>
       </ul>
-      Thank you
     </p>
     <p>
-      <%= link_to "LSA Ride Share", all_root_url %>
+      You can email psych.transportation@umich.edu or call the office at: 734-764-2580.
```

Comment:
This needs to be a per unit message. using the Unit_preferences would be a better solution.

Reply:
done in [37fb120](https://github.com/lsa-mis/lsa_ride_share/pull/160/commits/37fb1207bf4b24c6abf96c8892dd6efc45e5cff7)